### PR TITLE
Add description for bwein-net/contao-bfv-elements

### PR DESCRIPTION
**Pull request by rabauss**
P.S. The linter/allowlists are no longer fully sorted alphabetically ;-)

### CODE REVIEW

File: meta/bwein-net/contao-bfv-elements/de.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+de:
+    title: BFV Elemente
+    description: >
+        Diese Erweiterung bietet Inhaltselemente für die Bayerischer Fussball Verband Widgets (https://www.bfv.de/).
```
Comment (xchs) [inline meta/bwein-net/contao-bfv-elements/de.yml:4]:
```suggestion
        Diese Erweiterung bietet Inhaltselemente für die Bayerischer Fußball Verband Widgets (https://www.bfv.de/).
```

File: linter/allowlists/default.txt [hunk 2]
```diff
@@ -85,6 +87,7 @@ Fred
 Freunde
 Front end
 front end
+Fussball
```
Comment (xchs) [inline linter/allowlists/default.txt:90]:
```suggestion
```

File: meta/bwein-net/contao-bfv-elements/de.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        Das Blockieren von Cookies aus den Widgets wird über die Erweiterung "Contao Cookiebar" unterstützt.
+    keywords:
+        - contao
+        - bfv
+        - bayern
+        - bayerischer
+        - fussball
```
Comment (xchs) [inline meta/bwein-net/contao-bfv-elements/de.yml:11]:
```suggestion
        - fußball
```

File: meta/bwein-net/contao-bfv-elements/en.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+en:
+    title: BFV Elements
+    description: >
+        This Bundle provides content elements for the widgets of Bayerischer Fussball Verband (https://www.bfv.de/).
```
Comment (xchs) [inline meta/bwein-net/contao-bfv-elements/en.yml:4]:
```suggestion
        This Bundle provides content elements for the widgets of Bayerischer Fußball Verband (https://www.bfv.de/).
```

File: meta/bwein-net/contao-bfv-elements/en.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+        Blocking of cookies from the widgets is supported via the extension "Contao Cookiebar".
+    keywords:
+        - contao
+        - bfv
+        - bayern
+        - bayerischer
+        - fussball
```
Comment (xchs) [inline meta/bwein-net/contao-bfv-elements/en.yml:11]:
```suggestion
        - fußball
```

**Comment by aschempp** [comment]
Can you give a hint what this extension does?

**Comment by rabauss** [comment]
It's only a wrapper for the widgets of BFV so that you can integrate them as content elements and you don't need to use the script generator from the BFV

File: linter/allowlists/default.txt [hunk 6]
```diff
@@ -85,6 +87,7 @@ Fred
 Freunde
 Front end
 front end
+Fußball
```
Comment (xchs) [inline linter/allowlists/default.txt:90]: Did you get an error message from the linter if this keyword is not in the allowlist? I wonder why it is necessary to include the word "Fußball" here.

Comment (rabauss) [reply linter/allowlists/default.txt:90]: Yes my the linter complained about Fußball!

Comment (fritzmg) [reply linter/allowlists/default.txt:90]: But not for the German version, presumably? The word should only be added to the English allow list.

**Review by xchs — APPROVED** [review]

**Comment by aschempp** [comment]
> It's only a wrapper for the widgets of BFV so that you can integrate them as content elements and you don't need to use the script generator from the BFV

so it's something for the public (and not client-specific)?

**Comment by rabauss** [comment]
It is comparable with the youtube content element of the core - there you can add a ID of a Youtube video and the embed code is generated by the template.

That's the same for the widgets of BFV - see: https://www.bfv.de/der-bfv/digitalangebote/widgets/die-bfv-widgets

**Review by aschempp — APPROVED** [review]

**Pull request merged**